docs/src/adding_a_new_equation.md

```diff
@@ -0,0 +1,246 @@
+at a single physical location. Hence, we must use `u.^3` instead of the scalar
+operation `u^3`.
+
+That's already enough to run a simple simulation with a standard DGSEM discretization
+using the non-dissipative central flux at interfaces. This code is written outside
+of our new `module`.
+```julia
+# Create a simulation setup
+import .CubicConservationLaw
+using Trixi
+using OrdinaryDiffEq
+
+equation = CubicConservationLaw.CubicEquation()
+
+initial_condition_sine(x, t, equation::CubicConservationLaw.CubicEquation) = SVector(sinpi(x[1]))
+
+mesh = mesh = TreeMesh(-1.0, 1.0, # min/max coordinates
+                       initial_refinement_level=4,
+                       n_cells_max=10^4)
+
+solver = DGSEM(3 #= polynomial degree =#, flux_central)
+
+semi = SemidiscretizationHyperbolic(mesh, equation, initial_condition_sine, solver)
+
+# Create ODE problem with given time span
+tspan = (0.0, 0.09)
+ode = semidiscretize(semi, tspan)
+```
+We wrap the return value of the `initial_condition_sine` inside an `SVector` since that's the approach
+used in Trixi.jl also for systems of equations. We need to index the spatial coordinate `x[1]`,
+since it is an `SVector` with one component. In multiple space dimensions, all spatial coordinates
+are passed together.
+
+The `ode` is an `ODEProblem` from the SciML/DifferentialEquations ecosystem.
+Thus, we can solve this ODE numerically using any time integration method,
+e.g. `SSPRK43` from [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl).
+Before, we set up a [callback](@ref callbacks-id) to summarize the simulation setup.
+```julia
+summary_callback = SummaryCallback()
+callbacks = CallbackSet(summary_callback)
+
+# OrdinaryDiffEq's `solve` method evolves the solution in time and executes the passed callbacks
+sol = solve(ode, SSPRK43(),
+            save_everystep=false, callback=callbacks, maxiters=1e5);
+
+# Print the timer summary
+summary_callback()
+```
+That's it, you ran your first simulation using your new equation with Trixi! Now,
+we can plot the solution at the final time using Plots.jl.
+```julia
+plot(sol)
+```
+
+![tutorial_adding_new_equations_plot1](https://user-images.githubusercontent.com/12693098/111651488-91122980-8806-11eb-848c-af09f3af234c.png)
+
+You can already see that a discontinuities will develop and oscillations start to
+occur around steep parts of the wave. That's expected from our central discretization.
+To avoid these issues, we need to use dissipative numerical fluxes (approximate
+Riemann solvers) at interfaces.
+
+
+## Advanced setup
+
+Thus, we add a Godunov's flux for our cubic equation. That is easy for this equation
+since the wave speed `f'(u) = 3u^2` is always non-negative.
+```julia
+@inline Trixi.flux_godunov(u_ll, u_rr, orientation, equation::CubicEquation) = flux(u_ll, orientation, equation)
+```
+Let's run the example again but with a dissipative numerical flux at interfaces.
```

Comment:
```suggestion
Let's run the example again but with a dissipative numerical flux at interfaces.
[`remake`](@ref) will recreate the semidiscretization we used before and only change
selected parameters, in this case the `solver`.
```

Reply:
We don't have a docstring for `remake` in our docs (since it comes from DiffEqBase and is already described with enough information there). Thus, `[`remake`](@ref)` will throw an error. But I added the other changes in 550063bb.